language/move-prover/spec-lang/src/env.rs

```diff
@@ -857,6 +869,24 @@ impl GlobalEnv {
             .iter()
             .for_each(|(l, i)| f(l, i))
     }
+
+    /// Returns true if the boolean property is true.
+    pub fn is_property_true(&self, properties: &PropertyBag, name: &str) -> Option<bool> {
```

Comment:
Is this "property" something like the attribute mechanism in Boogie?  What things can have properties attached to them?

Reply:
Pretty much the same. Currently it can appear in form of a `pragma` at functions and in conditions (`<cond> [prop] <predicate>`)